style.css

```diff
@@ -0,0 +1,105 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&display=swap');
+
+body,
+html {
+  min-width: 100vw;
+  position: absolute;
+}
+
+header {
+  background: #3c3c3c;
+  margin: 0;
+  display: flex;
+  justify-content: space-between;
+  height: 70px;
+}
+
+header a {
+  font-size: 20px;
+  font-style: normal;
+  font-weight: bold;
+  color: rgb(255, 255, 255);
+  margin-left: 6%;
+  align-self: flex-start;
+  margin-top: auto;
+  margin-bottom: auto;
+  cursor: pointer;
+  text-decoration: none;
+}
+
+button {
+  margin-right: 6%;
+  margin-bottom: auto;
+  align-self: flex-end;
+  margin-top: auto;
+}
```

Comment:
- [ ] Your project looks great so far! 🎉  Its just the nav button does not match the Figma template. It looks like you need to add a few style properties (such as `background: transparent; border: none;`) in the button css to match it with the Figma design 👍🏽 🙂 

**_Figma Design_ 👇🏽** 
<img width="304" alt="Screenshot 2022-12-01 at 10 45 50 PM" src="https://user-images.githubusercontent.com/24832846/205117406-460b5fbc-0893-4373-bb32-9b49f432d383.png">

**_Your Design 👇🏽_** 
<img width="372" alt="Screenshot 2022-12-01 at 10 47 10 PM" src="https://user-images.githubusercontent.com/24832846/205117726-38d0f9e6-7df8-4e6a-99d6-a2bfc5c930ac.png">